wondrous-app/components/Common/Sidebar/Entity/AboutEntity/SettingsButton.tsx

```diff
@@ -0,0 +1,31 @@
+import { ButtonBase } from '@mui/material';
+import styled from 'styled-components';
+
+const Settings = styled(ButtonBase)`
```

Comment:
can we use a styles file for this?

Reply:
Can we keep the styles because the file is still only a few lines long?